### PR TITLE
feat: show launch plan information in workflow's schedules

### PR DESCRIPTION
**Pull request by pradithya**
# TL;DR

Add information of launch plan name and version to the workflow's schedule. 

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description

When there are multiple scheduled launchplan bound to a workflow, it is hard to understand which schedule correspond to which launchplan. This PR add additional information in the workflow's schedule to include the launch plan name and version. Additionally, when users clicked the launch plan name it will open the launchplan details page. 

### Before

Screenshot 2023-04-01 at 3 28 50 PM

### After
Screenshot 2023-04-01 at 3 32 09 PM

### Transition

https://user-images.githubusercontent.com/4023015/229272631-5c388db0-35ef-48da-9094-73f42e9ca4dd.mov

## Tracking Issue

fixes [https://github.com/flyteorg/flyte/issues/3554](https://github.com/flyteorg/flyte/issues/3554)

## Follow-up issue
_NA_

### CODE REVIEW

File: packages/console/src/components/Entities/EntitySchedules.tsx [hunk 1]
```diff
@@ -25,26 +35,61 @@ const useStyles = makeStyles((theme: Theme) => ({
...
-        const offsetString = getScheduleOffsetString(schedule);
-        const scheduleString = offsetString
-          ? `${frequencyString} (offset by ${offsetString})`
-          : frequencyString;
-        return (
-          <li key={identifierToString(launchPlan.id)}>{scheduleString}</li>
-        );
-      })}
-    </ul>
+    <TableContainer component={Paper}>
```
Comment (jsonporter) [inline packages/console/src/components/Entities/EntitySchedules.tsx:48]: Let me sync with our UX team on this; we might want to also show which entity version which schedule is applied to.

**Review by jsonporter — APPROVED** [review]
